pkg/agent/flowexporter/connections/connections.go

```diff
@@ -190,7 +192,7 @@ func (cs *ConnectionStore) Poll() (int, error) {
 	// We do not expect any error as resetConn is not returning any error
 	cs.ForAllConnectionsDo(resetConn)
 
-	filteredConns, err := cs.connDumper.DumpFlows(openflow.CtZone)
+	filteredConns, totalConns, err := cs.connDumper.DumpFlows(openflow.CtZone)
```

Comment:
pretty minor comment, but I am not a big fan of the naming here, since we have `filteredConns`, `totalConns` and `maxConns`, but the first one is a list while the others are integers. I would suggest renaming the first one to `filteredConnsList` or renaming the others to `totalNumConns` and `maxNumConns`.

Reply:
Thanks, I renamed the `filteredConns` to `filteredConnsList`.